bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -285,6 +285,16 @@ default String getDataUrl() {
         return null;
     }
 
+    /**
+     * Returns the url to fetch custom functions ESM
```

Comment:
`Returns the url to fetch custom functions bundled as an ESM` might have been better.

Reply:
made the change in the string